scripts/create-kms-key.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+alias="crypt4gh"
```

Comment:
Eh... what? Don't overload the builtin command.

Reply:
That's just a badly named variable. :)
That `alias` is referring to a kms alias, not a system alias.